src/app/modules/workspace/workspace.controller.ts

```diff
@@ -301,3 +252,40 @@ export const moveFormsToWorkspace: ControllerHandler<
       return res.status(statusCode).json({ message: errorMessage })
     })
 }
+
```

Comment:
On L243 there's also the function name vs log `meta.action` mismatch. Should be `moveFormsToWorkspace` instead?

Reply:
[b734172](https://github.com/opengovsg/FormSG/pull/6754/commits/b734172b81001f24780c9a374ae0ee1db95bdf97) 🙏